src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,10 @@
+import { MovieCard } from '../MovieCard';
 import './MovieList.scss';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard movie={movie} />
```

Comment:
add a react `key` whenever you render a list of items

Reply:
sure, I forgot)